examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,187 @@
+            let viewerDiv = document.getElementById('viewerDiv');
+
+            // Define initial camera position :
+            let initialPlacement = {
+                coord: new itowns.Coordinates('EPSG:4326', 2.351323, 48.856712),
+                range: 50000
+            };
+
+            // Instantiate iTowns GlobeView :
+            let view = new itowns.GlobeView(viewerDiv, initialPlacement);
+
+            // Sets a loading screen :
+            setupLoadingScreen(viewerDiv, view);
+
+            // Add one imagery layer to the scene and the miniView.
+            // This layer is defined in a json file but it could be defined as a plain js
+            // object. See Layer for more info.
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                const orthoLayer = new itowns.ColorLayer('Ortho', config);
+                view.addLayer(orthoLayer);
+            });
+
+            // Add two elevation layers.
+            // These will deform iTowns globe geometry to represent terrain elevation.
+            function addElevationLayerFromConfig(config) {
+                config.source = new itowns.WMTSSource(config.source);
+                let layer = new itowns.ElevationLayer(config.id, config);
+                view.addLayer(layer);
+            }
+            itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
+            itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
+
+
+            // THREE.Group which stores picked points markers.
+            let pickedPointsGroup = new itowns.THREE.Group();
+            pickedPointsGroup.visible = false;
+            view.scene.add(pickedPointsGroup);
+
+            // Add a pointer at a given position.
+            function addMeshToScene(position) {
+                // Copy the view camera3D to modify its far plane distance.
+                // The will render a CameraHelper with shorter axes.
+                const copyCamera = view.camera.camera3D.clone();
+                copyCamera.far = 50;
+                // Creation of the new mesh :
+                const mesh = new itowns.THREE.CameraHelper(copyCamera);
+                mesh.lookAt(position);
+                mesh.updateMatrixWorld();
+                // Add mesh to scene :
+                pickedPointsGroup.add(mesh);
+                view.notifyChange();
+            }
+
+            // Remove all meshes from scene.
+            function clearMeshes() {
+                pickedPointsGroup.remove(...pickedPointsGroup.children);
+                view.notifyChange();
+            }
+
+            // Array which stores picked camera positions, headings, tilts...
+            let cameraTravel = [];
+            // Camera speed in meters per second
+            let cameraSpeed = 500 / 3.6;
```

Comment:
may be use same time between each position, WDYM?

Reply:
Using same time between each position would result in camera movement being quite irregular. Movements with both long and short distance steps would suffer from short movements taking too much time or long movements being to fast (according to the value of the single time span of each step).

Reply:
Moreover, having a time for each step proportional to the distance of the step allows having a constant speed camera.

Reply:
May be add camera speed in `cameraUtils` pameters in another PR?